### PR TITLE
General improvements and fixes

### PR DESCRIPTION
**Pull request by 7sne**

### CODE REVIEW

**Review by krzkaczor — APPROVED** [review]

File: pages/unix-epoch-utc-conversion.test.tsx [hunk 1]
```diff
@@ -61,15 +59,6 @@ async function expectUtcError(
 }
 
 describe(UnixEpochUtcConversion.name, () => {
-  it('displays correct unix epoch time', async () => {
```
Comment (krzkaczor) [inline pages/unix-epoch-utc-conversion.test.tsx:64]: why delete test case?

File: src/misc/schemas/binarySchema.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { z } from 'zod';
 
-export const binarySchema = z.string().regex(/\b[01]+\b/, {
+export const binarySchema = z.string().regex(/^[01]*$/, {
```
Comment (krzkaczor) [inline src/misc/schemas/binarySchema.ts:3]: are there any tests to cover this change?

Comment (7sne) [reply src/misc/schemas/binarySchema.ts:3]: It's basically a more concise way to write the same regexp, nothing really changed, but if smth. would go wrong, there are tests against it.

File: pages/unix-epoch-utc-conversion.test.tsx (hunk 1 above)
Comment (7sne) [reply pages/unix-epoch-utc-conversion.test.tsx:64]: I decided that this test was not useful at all. To make it work we need: either mock the fn. returns (so it loses sense here), or match it against a range, I find it rather obscure.

**Pull request merged**